code/style.css

```diff
@@ -1,7 +1,423 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+@import url("https://fonts.googleapis.com/css2?family=East+Sea+Dokdo&family=Noto+Serif:wght@400;700&family=Open+Sans&display=swap");
 
-.active + .answer {
-  /* some styling here */
+/*** MAIN STYLING ***/
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Open sans", sans-serif;
+  line-height: 1.5;
+  font-size: 1rem;
+  background: #0e337d;
+  color: #cfe6ad;
+}
+
+a {
+  text-decoration: none;
+}
+
+h1 {
+  font-size: 3rem;
+  font-weight: 300;
+  line-height: 1.2;
+}
+
+h2 {
+  font-size: 2.5rem;
+}
+
+h3 {
+  font-size: 1.6rem;
+}
+
+/*** HEADER ***/
+
+.hero-section {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  color: #fff;
+  padding: 0 20px;
+}
+
+.hero-video {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+
+.hero-video video {
+  min-width: 100%;
```

Comment:
This is making your content too wide for mobile:
<img width="359" alt="Skärmavbild 2021-02-16 kl  14 43 49" src="https://user-images.githubusercontent.com/51529201/108070926-639a5a80-7065-11eb-92bd-e534e4c9ebf4.png">

Reply:
Fixed! :)